components/tikv_util/src/config.rs

```diff
@@ -34,14 +34,18 @@ pub enum ConfigError {
 }
 
 const UNIT: u64 = 1;
-const DATA_MAGNITUDE: u64 = 1024;
-pub const KB: u64 = UNIT * DATA_MAGNITUDE;
-pub const MB: u64 = KB * DATA_MAGNITUDE;
-pub const GB: u64 = MB * DATA_MAGNITUDE;
-
-// Make sure it will not overflow.
-const TB: u64 = (GB as u64) * (DATA_MAGNITUDE as u64);
-const PB: u64 = (TB as u64) * (DATA_MAGNITUDE as u64);
+const DECIMAL_DATA_MAGNITUDE: u64 = 1000;
```

Comment:
Is it being used? How about removing them to ensure no existing code is accidentally referencing the old name?

Reply:
Good idea.